### DISCUSSION TITLE
Głosowanie na sesje na 46 spotkanie Microsoft Azure User Group Poland

### DISCUSSION THREAD

**Discussion by aryll** [AzureCommunityPL/group#261] [Wydarzenia]
46 meetup już zaraz po świętach - 14 kwietnia o godzinie 18.00 :bangbang:

Chcesz mieć wpływ to zagłosuj na zgłoszone sesje :sunglasses:
Głosowanie do 8.04 do 23:00 :bangbang:

https://azurecommunitypl.typeform.com/to/uydmpGpv

![119995503_3247993851975298_3235021691368732427_n](https://user-images.githubusercontent.com/5950108/113395711-8f19a000-939a-11eb-9fb3-1c84efada4bb.png)

**Comment by aryll** [comment]
Sesje wybrane! Dzięki za głosy!
![image](https://user-images.githubusercontent.com/5950108/114146098-791a5b00-9917-11eb-849c-7d7db1ffedbb.png)

**Reply by KamilWiecek** [reply]
Rozpocząłem serię wpisów na blogu powiązanych z moim wystąpieniem. Zaczynamy od określenia co jest potrzebne, żeby prowadzić projekt w Azure (https://www.kamilwiecek.com/blog/landing-zone-as-a-service).  Skończymy (za kilka/ kilkanaście tygodni) z systemem self-service, który utworzy to wszystko w kilka minut, na życzenie.

**Comment by piotrgwiazda** [comment]
Trochę przypomina https://github.com/aztfmod/terraform-azurerm-caf który ma
swoje wady i zalety. Bardzo chętnie zobaczę co powstanie.

wt., 4 maj 2021, 21:30 użytkownik Kamil ***@***.***> napisał:
